e2e/test_module.py

```diff
@@ -0,0 +1,97 @@
+from pathlib import Path
+
+import pytest
+import yaml
+from .database import Memgraph, Node
+
```

Comment:
Chose a convention for importing and stick with it, e.g. first do all imports `import xy`  then 'import from', like you did in other files